restcomm/restcomm.interpreter/src/main/java/org/restcomm/connect/interpreter/VoiceInterpreter.java

```diff
@@ -399,6 +410,9 @@ public VoiceInterpreter(final Configuration configuration, final Sid account, fi
         this.fallbackMethod = fallbackMethod;
         this.statusCallback = statusCallback;
         this.statusCallbackMethod = statusCallbackMethod;
+        this.referUrl = referUrl;
+        this.referMethod = referMethod;
+        this.referTarget = referTarget;
```

Comment:
@vmykhailiuk referTarget should be in the download parameters (method parameters()) to be send to application server. This way the application server can create RCML that dials the transfer target `<Dial>${referTarget}</Dial>`

@otsakir can you please add this as a variable at RVD?